gnomad_qc/v3/annotations/generate_sv_hists.py

```diff
@@ -137,6 +138,7 @@ def get_sex_and_autosome_mt() -> hl.MatrixTable:
         reference_genome="GRCh38",
         min_partitions=300,
         force_bgz=True,
+        array_elements_required=False,
```

Comment:
why is this required now (and not previously)?

Reply:
Sorry -- this was a runtime error, didnt come up in my small test dataset but there was a missing value in the larger full dataset, and I fixed it locally for the original run.